src/Service/ProgrammesAggregationService.php

```diff
@@ -26,23 +31,56 @@ public function __construct(
     }
 
     /**
-     * @return Programme[]
+     * @return Series[]
      */
-    public function findProgrammesChildrenByType(
+    public function findDescendantSeries(
```

Comment:
Do we actually do this for lists of Series in frontend? http://www.bbc.co.uk/programmes/b006q2x0/series only shows immediate child series, not from all descendants.